reporter/html.js

```diff
@@ -723,6 +727,10 @@ QUnit.testDone(function( details ) {
 		time.innerHTML = details.runtime + " ms";
 		testItem.insertBefore( time, assertList );
 	}
+
+	sourceName.innerHTML = "<b class='counts'>Source: </b>" + details.source;
+	assertList.parentNode.insertBefore(sourceName, assertList.nextSibling);
```

Comment:
I was playing with the code and we can do this easier:

```js
        sourceName = document.createElement( "p" );
	sourceName.innerHTML = "<strong>Source: </strong>" + details.source;
	addClass( sourceName, "qunit-collapsed qunit-source" );
	addEvent( testTitle, "click", function() {
		toggleClass( sourceName, "qunit-collapsed" );
	});
	testItem.appendChild( sourceName );
```

- The source can be inserted to any test
- it shouldn't use `counts` class, but just a strong tag on the `Source: ` text.
- the source element now uses a `qunit-source` class.
- the click event is made individually, this way you can still have the source information on skipped tests
- it is appended on the testItem element.